.circleci/config.yml

```diff
@@ -5,48 +5,66 @@ defaults: &defaults
     - image: canonicalwebteam/dev
 
 jobs:
+  lint-python:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Lint python code
+          command: yarn lint-python
+  lint-scss:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn
+      - run:
+          name: Lint sass
+          command: yarn lint-scss
   test-site:
     <<: *defaults
     steps:
       - checkout
       - run:
           name: Install dependencies
-          command: yarn && bundle install --path vendor/bundle
+          command: yarn && pip3 install -r requirements.txt
       - run:
           name: Build the site resources
           command: yarn run build
       - run:
           name: Run the site server
-          command: bundle exec jekyll serve -P 80 -H 0.0.0.0
+          command: ./entrypoint 0.0.0.0:80
           background: true
       - run:
           name: Check site is accessible
-          command: sleep 1 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://localhost
-  lint-nginx:
+          command: sleep 3 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://localhost
+  test-python:
     <<: *defaults
     steps:
       - checkout
       - run:
-          name: Install gixy
-          command: pip3 install gixy
+          name: Yarn install
+          command: yarn install
       - run:
-          name: Lint nginx configuration files
-          command: gixy nginx.conf
-  lint-scss:
-    <<: *defaults
-    steps:
-      - checkout
+          name: Yarn build
+          command: yarn build
+      - run:
+          name: Install requirements
+          command: pip3 install -r requirements.txt
       - run:
           name: Install dependencies
-          command: yarn
+          command: pip3 install coverage
       - run:
-          name: Lint sass
-          command: yarn lint-scss
-
+          name: Run tests with coverage
+          command: |
+            coverage run  --source=. -m unittest discover tests
+            bash <(curl -s https://codecov.io/bash) -cF python
 workflows:
   version: 2
   build:
     jobs:
-      - test-site
-      - lint-nginx
```

Comment:
Did you remove `lint-nginx`? Do you think we don't need it?

Reply:
This project wasn't set up on CircleCI and this was the first time I saw this check. I thought this test was an older one or something. Also is it possible to make changes in the nginx.conf? I thought we were using the deployment configs for this site.

Should I add it back? I was also confused by the command `gixy nginx.conf` Do we have an nginx.conf file in that path?

Reply:
Ah you're right. We use `nginx.conf`s in our static sites - e.g. https://github.com/canonical-web-and-design/cloud-init.io/blob/master/nginx.conf - and then we use Gixy to test them.

But you're right, this is now a Flask project and doesn't include a, `nginx.conf`. I'll remove that requirement and merge.